dataflow.txt

```diff
@@ -0,0 +1,20 @@
+                              ************
+                              * Endorser *
+                              ************
+                                    |
+                  Evidence          |Endorsements
+                 Appraisal----.     |
+                  Policy      |     |             Attestation
```

Comment:
Who is speaking the Appraisal Policy?

Reply:
Predominantly, it is the local organization/domain-administrator.  Administrative-Domain. Or Owner.

Reply:
Endorsement is a kind of manifest, like for a ship (a shipper!)